networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/gossip/AggregateGossipManagerTest.java

```diff
@@ -40,8 +40,10 @@
   private final GossipNetwork gossipNetwork = mock(GossipNetwork.class);
   private final GossipEncoding gossipEncoding = GossipEncoding.SSZ_SNAPPY;
   private final TopicChannel topicChannel = mock(TopicChannel.class);
-  private final GossipedAttestationConsumer gossipedAttestationConsumer =
-      mock(GossipedAttestationConsumer.class);
+
+  @SuppressWarnings("unchecked")
+  private final GossipedOperationConsumer<ValidateableAttestation> gossipedAttestationConsumer =
+      mock(GossipedOperationConsumer.class);
```

Comment:
There's an `@Mock` annotation you can use that works with generics:
```suggestion
  @Mock
  private GossipedOperationConsumer<ValidateableAttestation> gossipedAttestationConsumer;
```

If you go with this, you need to annotate the test class with `@ExtendWith(MockitoExtension.class)` I think.

Reply:
Apparently to be able to use `MockitoExtension.class`we need to import 
`org.mockito:mockito-junit-jupiter`. Not sure if it's worth the effort since `@SupressWarning("unchecked")` serves basically the same purpose in the testing code and is fewer lines of code.